### ISSUE TITLE
Quest: Filling the Soul Gem

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#57]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Strangletorn Vale / Ruins of Aboraz / NPC: Yenniku / Quest: Filling the Soul Gem
Link: https://www.wowhead.com/wotlk/quest=593/filling-the-soul-gem

Progress:  no playback.

Completion: Despite the voice-over is perfect, the text is a description of actions. The narrator voice is more suitable in this case, instead of the male troll voice.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
